docs/user-guide/advanced/replicate_evaluations.md

```diff
@@ -140,6 +137,18 @@ IN_FEATURES=1024 \
 eva predict_fit --config configs/vision/pathology/offline/<task>.yaml
 ```
 
+### H-optimus-0 (Bioptimus) - ViT-G14 [[5]](#references)
+Bioptimus released their H-optimus-0 which was trained on a collection of 500,000 H&E slides. The model weights
```

Comment:
link `Bioptimus` to https://www.bioptimus.com/?

Reply:
done